lab_6_pipeline/pipeline.py

```diff
@@ -107,6 +190,9 @@ def to_conllu(self, article: Article) -> None:
         Args:
             article (Article): Article containing information to save
         """
+        with open(article.get_file_path(kind=ArtifactType.UDPIPE_CONLLU),
+                  'w', encoding='utf-8') as f:
+            f.write(article.get_conllu_info())
```

Comment:
how do you ensure that there is a newline at the end of conllu file content?

Reply:
Well, this method is called if we previously used our analizer analyze() method in pipeline.run(). Analyze method adds the missing newline manually, how I described before, so we either have the newline or have nothing to convert to conllu at all and don't call to_conllu() method.